ekncontent/ekncontent/eknc-domain.c

```diff
@@ -891,7 +1022,7 @@ eknc_domain_query_finish (EkncDomain *self,
   QueryState *state = g_task_get_task_data (task);
   return g_object_new (EKNC_TYPE_QUERY_RESULTS,
                        "upper-bound", state->upper_bound,
-                       "models", state->models,
+                       "models", ptr_array_to_list_deep (state->models),
```

Comment:
Hmm, it doesn't seem to immediately work. GObject-Introspection doesn't complain and I'm able to pass in a GPtrArray into the property just fine. However, reading it out just gets an empty array (it seems like `get_property` is not even called).

I can move this into a separate patch and we can inspect as a low-priority thing there I suppose.